### DISCUSSION TITLE
Export only messages with attachments

### DISCUSSION THREAD

**Discussion by Flustik24** [Tyrrrz/DiscordChatExporter#861] [Q&A] [answered]
Hello, Is it possible to export only those messages that have attachments? For example, there is a channel with 10 messages, and only two have attachments, and that the program skips those messages that do not have attachments, and as a result after export there were only two messages with attachments.

**Comment by 96-LB** [comment, marked as answer]
Use the `has:file` [filter](https://github.com/Tyrrrz/DiscordChatExporter/wiki/Message-filters).
